app/models/opensearch.rb

```diff
@@ -111,9 +111,34 @@ def matches
     match_single_field_nested(:subjects, m)
 
     match_geodistance(m) if @params[:geodistance].present?
+    match_geobox(m) if @params[:geobox].present?
     m
   end
 
+  # https://opensearch.org/docs/latest/query-dsl/geo-and-xy/geo-bounding-box/
+  def match_geobox(match_array)
+    match_array << {
+      bool: {
+        must: {
+          match_all: {}
+        },
+        filter: {
+          geo_bounding_box: {
+            'locations.geoshape': {
+              top: @params[:geobox][:max_latitude],
+              bottom: @params[:geobox][:min_latitude],
+              left: @params[:geobox][:min_longitude],
+              right: @params[:geobox][:max_longitude]
+            }
+          }
+        }
+      }
+    }
+  end
+
+  # https://www.elastic.co/guide/en/elasticsearch/reference/7.17/query-dsl-geo-distance-query.html
```

Comment:
Weird that they haven't created a page for this type of search, but I see what you're talking about.

Reply:
I suspect the Docs licensing didn't allow a fork so I believe OpenSearch had to throw away all the docs and start over whereas with the code they had the fork... so docs required _a lot_ of work to build back up.